roles/nginxplus/files/conf/http/dev/templates/libnet.conf

```diff
@@ -0,0 +1,7 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
I have no idea what this file is or what it does or how it's related (or not) to the work of building Solr or backing up Solr indexes.